styleguide/css/MODERN-CSS.md

```diff
@@ -0,0 +1,138 @@
+![Barsoom](http://barsoom.se/barsoom.png)
```

Comment:
We don't appear to be linking to this file anywhere.

I'm also not sure about having two CSS styleguides or using the term "modern" to distinguish them. Let's discuss this later per other comment.

Reply:
We need separate styleguides for `app/assets` and `app/webpacker_assets`. `app/assets` should be considered legacy.